templates/pages/checkout.html

```diff
@@ -1 +1,30 @@
-brand.html
+{{#partial "head"}}
+
+{{{ checkout_head }}}
+{{{ stylesheet '/assets/css/optimized-checkout.css' }}}
+{{ getFontsCollection }}
+
+{{/partial}}
+
+{{#partial "page"}}
+<header class="checkoutHeader optimizedCheckout-header">
+    <div class="checkoutHeader-content">
+        <h1 class="is-srOnly" translate>Checkout</h1>
+        <h2 class="checkoutHeader-heading">
+            <a class="checkoutHeader-link" href="{{urls.home}}">
+                {{#if HeaderImage }}
+                        <img alt="{{settings.store_logo.title}}" class="checkoutHeader-logo" id="logoImage" src="{{ HeaderImage }}"/>
```

Comment:
I think we should follow the same convention as other Stencil objects (i.e.: https://stencil.bigcommerce.com/docs/cart-resources). Instead of having a global `HeaderImage`, nest it under `checkout`. i.e.: `checkout.header_image`.